simulation/README.md

```diff
@@ -32,6 +33,11 @@ simulation:
 
 `data_file`: This is the file where all recorded events are stored.
 
+`callers`: is a list of event sources for recording of events. When an item is changed, the change is done
+by someone, e.g. knx for changes from the bus. The canner name is identical with the plugin name that changes the item.

```

Comment:
Hi,
You are right. The programmer sets the caller name. Even I did it in the plugin :-)
Just did not remember.
I corrected the description. 

Michael